windows/src/engine/kmcomapi/util/utilkeyman.pas

```diff
@@ -275,9 +275,33 @@ function PackageInstalled(const PackageName: string; var FIsAdmin: Boolean): Boo
   end;
 end;
 
+function GetKeyman32Name: string;
```

Comment:
This function is duplicated in keymancontrol.pas

Reply:
Only did half the job of moving it out to util.